packages/polaris-typeorm/src/typeorm-bypasses/polaris-entity-manager.ts

```diff
@@ -192,7 +199,7 @@ export class PolarisEntityManager extends EntityManager {
                 undefined,
                 true,
             ).getRawMany();
-            const result: { entityId: string; maxDV: number }[] = this.getIdsAndTheirMaxDvs(
+            const result: { id: string; maxdv: number }[] = this.getIdsAndTheirMaxDvs(
```

Comment:
in data version handler it runs on entityid alias sometimes, check it still works

Reply:
its fine, i do it after the query finish execution. all the `entityid` alias happens inside the data version handler before the query execution.